extensions/parquet/base/src/main/java/io/deephaven/parquet/base/ColumnChunkReader.java

```diff
@@ -32,9 +34,24 @@ public interface ColumnChunkReader {
      */
     int getMaxRl();
 
+    /**
+     * @return The offset index for this column chunk, or null if it not found in the metadata.
+     */
+    @Nullable
+    OffsetIndex getOffsetIndex();
+
     interface ColumnPageReaderIterator extends Iterator<ColumnPageReader>, AutoCloseable {
```

Comment:
Should I rename this class to `ColumnPageReaderAccessor` because now there are two ways to create a ColumnPageReader for a particular page, we can either iterate and get it from `next()` or we can directly create one using the page number?

Reply:
Done